providers/apis/dydx/research_api_handler.go

```diff
@@ -136,12 +140,16 @@ func (h *ResearchAPIHandler) ParseResponse(
 
 // researchJSONToQueryAllMarketsParamsResponse converts a dydx research json into a
 // QueryAllMarketsParamsResponse.
-func researchJSONToQueryAllMarketsParamsResponse(research dydxtypes.ResearchJSON) (dydxtypes.QueryAllMarketParamsResponse, error) {
+func (h *ResearchAPIHandler) researchJSONToQueryAllMarketsParamsResponse(research dydxtypes.ResearchJSON) (dydxtypes.QueryAllMarketParamsResponse, error) {
 	// iterate over all entries in the research json + unmarshal it's market-params
 	resp := dydxtypes.QueryAllMarketParamsResponse{}
 	for _, market := range research {
+		if market.CMCID < 0 {
```

Comment:
Does this limit us (even the normal research-json provider) to ignore any market that does not have a cmc-id?

Reply:
good catch, should also check that its cmc